runtime/build.sh

```diff
@@ -4,7 +4,8 @@
 
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
-cargo build \
+ARBITRARY_MAX_HANDLES=32 cargo build \
```

Comment:
It might be easier to just hard code this to 32 for now and set the environment variable in the RT build script.

I think how this is written right now, all the tests are still using the default 24. And I'm not sure we really want to test both configurations.

Reply:
Hi, Jordan. Do you mean hardcode in caliptra-dpe?